### DISCUSSION TITLE
Only react to a giveaway if it has > (number)

### DISCUSSION THREAD

**Discussion by philhk** [marioparaschiv/nitro-sniper#145] [Ideas]
Many servers test you in a certain way: one person on the server gives a giveaway and then says that nobody should react to it and a bot cannot know that. So if you were to add a feature that the bot would only react to the giveaway after a certain amount of people react to it, the bot would be safe because no body would do it because they were told not to do it .

**Comment by InimicalPart** [comment]
The bot does know that's, that's why the blacklisted words are there.


> If you liked my response, let me know by reacting with a heart ❤️ Have a nice day, fella! :)

**Reply by philhk** [reply]
Yes, but people purposely put the word nitro in their giveaway (like a real giveaway) and then send a message themselves saying not to respond and the bot can't know than :/ It would be possible, but I think it would be a lot more complicated than my idea.

**Reply by InimicalPart** [reply]
True, but if you think about it, you don't really know if the limit will be good for the number of people on the server. The server members can be like 100 and your thing is set to 50, it's hard to get half of the server to react to a giveaway. You also don't know the time of the giveaway, what if it's like 30 seconds. Users will not hit that amount in such little time, making your sniper unefficient. 

> If you liked my response, let me know by reacting with a heart ❤️ Have a nice day, fella! :)
